pages/contact-us.vue

```diff
@@ -10,55 +10,149 @@
             {{ t("aboutUsDesc") }}
           </p>
         </div>
-
-        <div class="mt-8">
-          <div class="mt-8">
+      </div>
+    </div>
+    <div class="mt-20 flex gap-10 justify-between">
+      <div class="flex-1 mt-10 text-lg">
+        <h1 class="text-4xl font-bold font-[Poppins] pb-5">Contact Us</h1>
```

Comment:
font poppins semua di hapus dulu we jul di semua tempat. Nanti dibuat di satu tempat class ameh lebih gampang di ubah kalau di beberapa tempat. Kalo di satu satu nanti capek mun refactor